src/i18n/locales/en-US.json

```diff
@@ -291,10 +291,10 @@
   "components.walletinit.createwallet.mnemonicshowscreen.mnemonicNote": "Please, make sure you have carefully written down your recovery phrase somewhere safe. You will need this phrase to use and restore your wallet. Phrase is case sensitive.",
```

Comment:
Please remove also `src/i18n/locales/`, except maybe `/src/i18n/locales/**/en-US.json` and `/src/i18n/locales/**/en-US.md` which are the only ones that are edited manually. The rest is generated automatically and shouldn't be processed by prettier

Reply:
not sure how to do this

Reply:
then let's just ignore the full `src/i18n/` folder, and `translations/` too 🙏

Reply:
done